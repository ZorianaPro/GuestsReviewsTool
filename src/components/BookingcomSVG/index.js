import React from 'react';

const BookingcomSVG = (props) => (
  <svg className="BookingcomSVG" {...props}>
    <use xlinkHref="#BookingcomSymbol" />
  </svg>
);

export default BookingcomSVG;
