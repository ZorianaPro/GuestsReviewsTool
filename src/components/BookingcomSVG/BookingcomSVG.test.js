import React from 'react';
import { shallow } from 'enzyme';
import BookingcomSVG from './';

describe('BookingcomSVG', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <BookingcomSVG />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<BookingcomSVG />)).toMatchSnapshot();
  });
});
