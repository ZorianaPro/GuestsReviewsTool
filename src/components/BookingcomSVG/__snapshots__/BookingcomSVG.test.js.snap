// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookingcomSVG renders as expected 1`] = `
<svg
  className="BookingcomSVG"
>
  <use
    xlinkHref="#BookingcomSymbol"
  />
</svg>
`;
